cmd/ingester/app/flags.go

```diff
@@ -106,14 +108,61 @@ func AddFlags(flagSet *flag.FlagSet) {
 		DefaultDeadlockInterval,
 		"Interval to check for deadlocks. If no messages gets processed in given time, ingester app will exit. Value of 0 disables deadlock check.")
 
+	// TODO: Remove deprecated flags after 1.11
+	flagSet.String(
+		DeprecatedKafkaConfigPrefix+SuffixBrokers,
+		DefaultBroker,
+		"Deprecated; The comma-separated list of kafka brokers. i.e. '127.0.0.1:9092,0.0.0:1234'")
```

Comment:
rather than explaining the meaning, I suggest we refer to the new flag that's replacing this one, i.e. "deprecated; replaced by xxx"